pom.xml

```diff
@@ -123,6 +123,7 @@
     <slf4j.version>1.7.16</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
     <hadoop.version>2.7.4</hadoop.version>
+    <miniKdc.version>3.2.0</miniKdc.version>
```

Comment:
BTW, `miniKdc` -> `minikdc`.

Reply:
> BTW, `miniKdc` -> `minikdc`. As you see, Spark-introduced properties are are camelcase like `codahale` and `htmlunit`. If needed, we may use `-`, but I prefer `minikdc` here in this case.

Thank you for your careful explanation.  Updated.